apps/cowswap-frontend/src/modules/application/containers/App/Updaters.tsx

```diff
@@ -36,16 +37,18 @@ import { SolversInfoUpdater } from 'common/updaters/SolversInfoUpdater'
 import { UserUpdater } from 'common/updaters/UserUpdater'
 
 export function Updaters() {
-  const { chainId, account } = useWalletInfo()
+  const { account } = useWalletInfo()
   const { tokenLists, appCode, customTokens, standaloneMode } = useInjectedWidgetParams()
   const onTokenListAddingError = useOnTokenListAddingError()
   const { isGeoBlockEnabled, isYieldEnabled } = useFeatureFlags()
   const tradeTypeInfo = useTradeTypeInfo()
   const isYieldWidget = tradeTypeInfo?.tradeType === TradeType.YIELD
   const cowAnalytics = useCowAnalytics()
+  const sourceChainId = useSourceChainId()
 
   return (
     <>
+      <MultiCallUpdater chainId={sourceChainId} />
```

Comment:
A new updater in order to make multicall library work with `sourceChainId` instead of wallet chainId.

Reply:
We need it for balances and allowances fetching.